lessons/pydata/pandas_groupby/index.ipynb

```diff
@@ -0,0 +1,3297 @@
+{
```

Comment:
<p>Jaký to má vlastně vztah k crosstab (je v EDA4)?</p><p> </p><p>Někde jsem se dočetl:</p><p> </p><blockquote>[...crosstab...] Its a tabular structure showing relationship between different variables. The Pandas crosstab and pivot has not much difference it works almost the same way. The only difference that I see after going through the source code is Crosstab works with Series or list of Variables whereas Pivot works with dataframe and internally crosstab calls pivot table function. So when you have list of data or a Series then you should use crosstab and if there is data available in a dataframe then you should go for pivot table.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/73/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='85'/>